AirSkyBoat/AirSkyBoat
# Incorporating Kazham & Mhaura vendor price variability by using Windurst fame

**_I affirm:_**


## Description
No matter what your fame is set too, Mhaura and Kazham vendor prices do not change nor does the price of items you sell to them.

## What does this pull request do?

The two functions SmallPacket0x085 and SmallPacket0x084 in packet_system.cpp which appraises and sells items does not interact in any way with Kazham or Mhaura vendor prices. This is because the function in GetFameAreaFromZone() in zoneutils.cpp doesn't mention Kazham or Mhaura. Adding them to the switch case with Windurst means that vendors will now use Windurst fame when calculating buy/sell prices.

Kazham and Maura seem to have a 1:1 fame relationship with Windurst. https://wiki.ffo.jp/html/2683.html

## Steps to test these changes
!setfamelevel 2 1 Player
Go to a vendor in the mentioned zones and check the price of an item and the sell price of one of your items.
!setfamelevel 2 9 Player
Check again, notice the difference
Checking a fame reputation reporter also verifies the fame changing.
